test-suite/misc/coq_environment.sh

```diff
@@ -9,14 +9,15 @@ cd $TMP
 cat > coq_environment.txt <<EOT
 # we override COQLIB because we can
 COQLIB="$TMP/overridden" # bla bla
+COQCORELIB="$TMP/overridden" # bla bla
```

Comment:
What does "bla bla" mean here? cc @gares

Reply:
It tests we can put comments